versions.sh

```diff
@@ -5,13 +5,15 @@ set -Eeuo pipefail
 declare -A otpMajors=(
 	[3.9]='24'
 	[3.10]='25'
+	[3.11]='25'
 )
 
 # https://www.openssl.org/policies/releasestrat.html
 # https://www.openssl.org/source/
 declare -A opensslMajors=(
 	[3.9]='1.1'
 	[3.10]='1.1'
+	[3.11]='3.0'
```

Comment:
@michaelklishin @gerhard any opinions on whether this is actually a sane release to upgrade to OpenSSL 3.0 on? :innocent:

(I figured I'd start with 3.0 to open the conversation / test the build, and let y'all tell me if I'm being too aggressive and should drop back down to 1.1 :+1:)

Reply:
We haven't gotten to use OpenSSL 3.0 by default yet. I am not trying to discourage you trying it, it's just we won't be of much help until at least next month.

Reply:
Oh, totally fair -- I'll swap this back to 1.1 for now (y'all being helpful/comfortable with it is worth it IMO :smile:).

Reply:
I'm glad I didn't spend much time on OpenSSL 3 here!  Looks like it's not until Erlang OTP 25.1 that it's even fully supported there (https://github.com/erlang/otp/releases/tag/OTP-25.1), so I was doubly jumping the gun. :joy: